README.md

```diff
@@ -1,2 +1,157 @@
 # example-python
 An example repo to demonstrate Python support in Pants v2.
+
+See https://pants.readme.io/ for much more detailed documentation.
```

Comment:
You might want to mention the specific page in the docs for turning on V2 mode. This won't work properly if they use Pants without the correct config.

Reply:
This repo is already configured correctly. the README is intended to reflect examples in this repo, not general-purpose documentation, so I want to keep it succinct and not entangle any such issues. The documentation I linked to is our new v2-only documentation, so readers there should have all the info they need.